docs/standard/exceptions/best-practices-for-exceptions.md

```diff
@@ -163,7 +163,12 @@ This example illustrates the use of `throw` to re-throw the original exception,
 catch (Exception ex)
 {
     from.RollbackTransaction(withdrawalTrxID);
-    throw new Exception("Withdrawal failed", ex);
+    throw new TransferFundsException("Withdrawal failed", ex)
```

Comment:
Would it make sense to use a named argument here to make the code clearer? I.e.:

```c#
throw new TransferFundsException("Withdrawal failed", innerException: ex)
```

I wouldn't expect that in regular code, but maybe it makes sense here?